kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/ZookeeperConf.scala

```diff
@@ -49,6 +49,13 @@ object ZookeeperConf {
       .stringConf
       .createOptional
 
+  val ZK_CLIENT_USE_HOSTNAME: ConfigEntry[Boolean] =
+    buildConf("kyuubi.zookeeper.embedded.client.use.hostname")
+      .doc("When true, embedded Zookeeper prefer to bind hostname, otherwise, ip address.")
+      .version("1.7.2")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
Should we keep the original logic and default set true to bind hostname?

Reply:
Ignore this, LGTM default to bind on ip, consider that in most cases(especially On k8s, to reduced DNS discovery), bind ip is better for bind hostname

Reply:
yea, was thought that, but since it's not for production, I think behavior change should be fine.